lib/services/messenger/translators/flowdock.ts

```diff
@@ -87,6 +88,7 @@ export class FlowdockTranslator extends TranslatorScaffold implements Translator
 			metadata.content = metadata.content.replace(new RegExp(findPublic, 'igm'), '').trim();
 			if (metadata.title) {
 				metadata.title = metadata.title.replace(new RegExp(findPublic, 'igm'), '').trim();
+				metadata.title = new AllHtmlEntities().decode(metadata.title);
```

Comment:
Maybe assign an instance to a module-local variable and reuse it to save the instantiation on each request/call?